test/unit/views/partials/BackLinkViewSpec.scala

```diff
@@ -55,128 +46,74 @@ class BackLinkViewSpec extends ViewSpec {
-      doc.getElementById("back-link") should haveAttribute("href", "url")
+      doc.getElementById("back-link") should haveAttribute("href", "#")
       doc.text()                      shouldBe "Back"
     }
 
     "render back link with details from session with custom text for search result" in {
-      val customLabel = "custom"
-      // Given
-      val requestWithSessionData = AuthenticatedRequest(
-        Operator("0", Some("name")),
-        FakeRequest().withSession(
-          (backToSearchResultsLinkUrl, "url"),
-          (backToSearchResultsLinkLabel, customLabel)
-        )
-      )
-
       // When
-      val doc = view(back_link()(requestWithSessionData, messages))
+      val doc = view(back_link()(messages))
 
       // Then
       doc                             should containElementWithID("back-link")
       doc.getElementById("back-link") should haveTag("a")
-      doc.getElementById("back-link") should haveAttribute("href", "url")
-      doc.text()                      shouldBe "Back to " + customLabel
+      doc.getElementById("back-link") should haveAttribute("href", "#")
+      doc.text()                      shouldBe "Back"
     }
 
     "render back link with details from session with default text for queues result" in {
-      // Given
-      val requestWithSessionData = AuthenticatedRequest(
-        Operator("0", Some("name")),
-        FakeRequest().withSession(
-          (backToQueuesLinkUrl, "url"),
-          (backToQueuesLinkLabel, "")
-        )
-      )
-
       // When
-      val doc = view(back_link()(requestWithSessionData, messages))
+      val doc = view(back_link()(messages))
 
       // Then
       doc                             should containElementWithID("back-link")
       doc.getElementById("back-link") should haveTag("a")
-      doc.getElementById("back-link") should haveAttribute("href", "url")
+      doc.getElementById("back-link") should haveAttribute("href", "#")
       doc.text()                      shouldBe "Back"
     }
 
     "render back link with details from session with custom text for queues result" in {
-      val customLabel = "custom"
-      // Given
-      val requestWithSessionData = AuthenticatedRequest(
-        Operator("0", Some("name")),
-        FakeRequest().withSession(
-          (backToQueuesLinkUrl, "url"),
-          (backToQueuesLinkLabel, customLabel)
-        )
-      )
-
       // When
-      val doc = view(back_link()(requestWithSessionData, messages))
+      val doc = view(back_link()(messages))
 
       // Then
       doc                             should containElementWithID("back-link")
       doc.getElementById("back-link") should haveTag("a")
-      doc.getElementById("back-link") should haveAttribute("href", "url")
-      doc.text()                      shouldBe "Back to " + customLabel
+      doc.getElementById("back-link") should haveAttribute("href", "#")
+      doc.text()                      shouldBe "Back"
     }
```

Comment:
I think all of these the tests in this spec are pretty much the same now, so perhaps we could get rid of all but one?

Reply:
Yep good point David, I'll update them now.

Reply:
Done